src/libraries/System.Runtime.Loader/tests/ApplyUpdateTests.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Nit: `AssemblyExtensionsTest.cs` may be a better name for this file. The convention is to use the same name for the file and the main type in the file.